api/count_min_sketch.md

```diff
@@ -0,0 +1,77 @@
+---
+api_name: count_min_sketch()
+excerpt: Aggregate data in a `count_min_sketch` for calculation of estimates
+license: community
+toolkit: true
+topic: hyperfunctions
+tags: [hyperfunctions, frequency, count min sketch]
+api_category: hyperfunction
+api_experimental: true
+hyperfunction_toolkit: true
+hyperfunction_family: 'frequency analysis'
+hyperfunction_subfamily: CountMinSketch
+hyperfunction_type: aggregate
+---
+
+import Experimental from 'versionContent/_partials/_experimental.mdx';
+
+# count_min_sketch() <tag type="toolkit" content="Toolkit" /><tag type="experimental-toolkit" content="Experimental" />
+Produces a [Count-Min Sketch][count-min-sketch] in the form of an aggregate that can be passed to the [`approx_count` function][approx-count] to estimate how many times a particular value has appeared in a column.
+
+```sql
+count_min_sketch(
+    values TEXT,
+    error DOUBLE PRECISION,
+    probability DOUBLE PRECISION,
+) RETURNS CountMinSketch
+```
+
+
+## Required arguments
+
+|Name|Type|Description|
+|-|-|-|
+|`values`|`TEXT`|Column to aggregate|
+|`error`|`DOUBLE PRECISION`|Error tolerance in estimate, calculated relative to the number of values added to the sketch|
+|`probability`|`DOUBLE PRECISION`|Probability such that error bounds will hold with 1-`probability`% of the time|
```

Comment:
Instead of expressing this as the opposite, maybe it would be clear to express it in terms of the error? I.e. probability that the error will fall outside of the specified bounds